.github/workflows/lint.yml

```diff
@@ -109,6 +109,7 @@ jobs:
         with:
           components: rustfmt
       - run: cargo fmt --all --check
+      - uses: codespell-project/actions-codespell@v2
```

Comment:
i'd add this as a separate pass, just because if i see `fmt` fail i usually don't bother checking why i just run `cargo +nighy fmt --all` and push

Reply:
set as a seperate task